src/content/tools/devtools/deep-links.md

```diff
@@ -0,0 +1,26 @@
+---
+title: Validate deep links
+description: Learn how to validate deep links in your app.
+---
+
+:::note
+The deep linking validator was added to DevTools
+in Flutter SDK 19. It currently works only for
+Android, but will be expanded to cover iOS and
```

Comment:
Just want to double check the next stable will include the app check right? @hangyujin